typo3Model.xml

```diff
@@ -41,6 +41,27 @@
       <type name="dkd:typo3:pages">
           <title>TYPO3 Page</title>
           <parent>cm:folder</parent>
+          <properties>
+              <property name="dkd:typo3:pages:doktype">
```

Comment:
Maybe this should also be dropped and should be done by inheritance. So a shortcut is a children of a page. This will help to define a cleat relationship